automation/capitanodoc/index.ts

```diff
@@ -57,11 +57,11 @@ function importCapitanoCommands(jsFilename: string): CapitanoCommand[] {
 	const commands: CapitanoCommand[] = [];
 
 	if (actions.signature) {
-		commands.push(_.omit(actions, 'action'));
+		commands.push(_.omit(actions, 'action') as any);
 	} else {
 		for (const actionName of Object.keys(actions)) {
 			const actionCommand = actions[actionName];
-			commands.push(_.omit(actionCommand, 'action'));
+			commands.push(_.omit(actionCommand, 'action') as any);
```

Comment:
This is due to updated lodash typings, previously because of `actionCommand` being `any` the result of the `_.omit` was automatically `any` but now it's a bit tighter and clashes with the `commands` typing